Reply/app/src/main/java/com/materialstudies/reply/ui/nav/BottomNavDrawerFragment.kt

```diff
@@ -87,16 +88,13 @@ class BottomNavDrawerFragment :
       mutableListOf()
 
     private val backgroundShapeDrawable: MaterialShapeDrawable by lazy(NONE) {
-        val backgroundContext = binding.backgroundContainer.context
+        val dynamicContext = DynamicColors.wrapContextIfAvailable(requireContext())
```

Comment:
I don't think you need to create this wrapped context. The backgroundContext that was being used should already have the DynamicColor theme overlay applied since it's a child of MainActivity